### PR TITLE
[backup] TransactionReplayer replays chunks with reconfigurations

### PR DESCRIPTION
**Pull request by msmouse**
## Motivation

Fix the issue that transaction restore fails when a chunk contains a reconfiguration.

### Have you read the [Contributing Guidelines on pull requests](https://github.com/libra/libra/blob/master/CONTRIBUTING.md#pull-requests)?
Y

## Test Plan

tests added

### CODE REVIEW

File: testsuite/smoke-test/src/smoke_test.rs [hunk 1]
```diff
@@ -2236,15 +2250,24 @@ fn db_restore(backup_path: &Path, db_path: &Path) {
...
     balance1: f64,
+    op_tool: OperationalTool,
+    backend: SecureBackend,
     quit: Arc<AtomicBool>,
 ) -> f64 {
     let now = Instant::now();
     let mut transferred = 0f64;
     while !quit.load(Ordering::Relaxed) {
+        if random::<u16>() % 10 == 0 {
+            let (txn_ctx, _) = op_tool.rotate_operator_key(&backend).unwrap();
```
Comment (msmouse) [inline testsuite/smoke-test/src/smoke_test.rs:2265]: .

**Review by lightmark — APPROVED** [review]

**Comment by davidiw** [comment]
cc: @JoshLind, maybe we should more quickly evaluate a shared storage solution....

File: libra-node/src/main_node.rs [hunk 2]
```diff
@@ -112,6 +112,7 @@ pub fn setup_environment(node_config: &NodeConfig, logger: Option<Arc<Logger>>)
     );
 
     let waypoint = node_config.base.waypoint.waypoint();
+    println!("WWWW1111: {}", waypoint);
```
Comment (msmouse) [inline libra-node/src/main_node.rs:115]: .. >_<

**Pull request closed without merging**